sdk/tables/azure-data-tables/azure/data/tables/aio/_table_client_async.py

```diff
@@ -37,19 +30,19 @@
 from ._base_client_async import AsyncTablesBaseClient
 from ._models import TableEntityPropertiesPaged
 from ._table_batch_async import TableBatchOperations
+from .._table_client import EntityType, TransactionOperationType
 
 
 class TableClient(AsyncTablesBaseClient):
     """ :ivar str account_name: Name of the storage account (Cosmos or Azure)"""
 
     def __init__(
         self,
-        account_url,  # type: str
-        table_name,  # type: str
-        credential=None,  # type: str
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> None
+        account_url: str,
+        table_name: str,
+        credential: Optional[Union[AzureSasCredential]] = None,
```

Comment:
`AzureSasCredential` or `NamedKeyCredential` right?

Reply:
Right - but the work to support NamedKeyCredential is outside the scope of this PR - so just leaving the typehint for what's supported today for now :)